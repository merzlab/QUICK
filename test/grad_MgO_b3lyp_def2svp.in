B3LYP  BASIS=def2-svp cutoff=1.0e-9 denserms=1.0e-6  zmake GRADIENT DIPOLE CHARGE=0

Mg        -6.41953        1.52486        0.00000
O         -4.60901        1.52486        0.00000


