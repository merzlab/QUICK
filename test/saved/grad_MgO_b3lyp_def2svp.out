  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Wed Mar 17 12:12:32 2021
| INPUT FILE :    grad_MgO_b3lyp_def2svp.in
| OUTPUT FILE:    grad_MgO_b3lyp_def2svp.out
| DATE FILE  :    grad_MgO_b3lyp_def2svp.dat
| BASIS SET PATH: /scratch/cruzeiro/quick/basis/

 @ Read Job And Atom

  KEYWORD=B3LYP  BASIS=DEF2-SVP CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE GRADIENT DIPOLE CHARGE=0                                                                                                                     
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 DENSITY FUNCTIONAL = B3LYP
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 STANDARD GRID = SG1
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
      GRADIENT CUTOFF  =  0.100E-08
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = DEF2-SVP,  TYPE = CARTESIAN
| BASIS FILE = /scratch/cruzeiro/quick/basis//DEF2-SVP.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = MG
 BASIS FUNCTIONS =   19
 For Atom Kind =    2
 ELEMENT = O 
 BASIS FUNCTIONS =   15

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    2    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    0    NUMBER OF NON-HYDROGEN ATOM =    2
 NUMBER OF ELECTRONS     =   20

 -- INPUT GEOMETRY -- :
    MG         -6.4195       1.5249       0.0000
    O          -4.6090       1.5249       0.0000

 -- DISTANCE MATRIX -- :
              1           2
      1     0.00000
      2     3.42139     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   34
 NSHELL =   14 NPRIM  =   62
 JSHELL =   14 JBASIS =   30


 @ End Reading Molecular Information 


 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-09
   INITIAL GRID POINTS  =        7576
|   FINAL GRID POINTS    =        5528
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        3584
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =        4191

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1   -275.070599950      ------         0.176     1        0.00      0.17      0.00    0.1074E+01  0.5427E-01  0.1004E+01
|  2   -274.683500065  -.387100E+00       0.163     2        0.00      0.16      0.00    0.5033E+00  0.9914E-01  0.1371E+01
|  3   -270.240045935  -.444345E+01       0.166     3        0.00      0.16      0.00    0.1581E+01  0.8981E-01  0.1094E+01
|  4   -275.007325702  0.476728E+01       0.166     4        0.00      0.16      0.00    0.2129E+00  0.2787E-01  0.5630E+00
|  5   -275.094721942  0.873962E-01       0.163     5        0.00      0.16      0.00    0.8025E-01  0.2273E-02  0.2907E-01
|  6   -275.097103307  0.238136E-02       0.135     6        0.00      0.13      0.00    0.7863E-01  0.9961E-03  0.2056E-01
|  7   -275.094495220  -.260809E-02       0.135     7        0.00      0.13      0.00    0.9223E-01  0.1858E-02  0.3698E-01
|  8   -275.100095251  0.560003E-02       0.135     8        0.00      0.13      0.00    0.5863E-01  0.2706E-02  0.5179E-01
|  9   -275.104075945  0.398069E-02       0.134     9        0.00      0.13      0.00    0.1057E-01  0.6883E-03  0.1121E-01
| 10   -275.104221287  0.145342E-03       0.134    10        0.00      0.13      0.00    0.9550E-03  0.1350E-03  0.1231E-02
| 11   -275.104223285  0.199781E-05       0.135     1        0.00      0.13      0.00    0.2220E-03  0.1472E-03  0.1434E-02
| 12   -275.104223802  0.516717E-06       0.134     2        0.00      0.13      0.00    0.1999E-03  0.2527E-04  0.2803E-03
| 13   -275.104223855  0.536600E-07       0.139     3        0.01      0.13      0.01    0.1334E-04  0.1022E-05  0.1015E-04
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
| 14   -275.104224198  0.342298E-06       0.138     4        0.00      0.14      0.00    0.9085E-06  0.7629E-07  0.8817E-06
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER  14 CYLCES
| MAX ERROR = 0.908474E-06   RMS CHANGE = 0.762901E-07   MAX CHANGE = 0.881701E-06
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =   10.0002580379
 BETA ELECTRON DENSITY     =   10.0002580379
 ELECTRONIC ENERGY    =  -303.163020579
 CORE_CORE REPULSION  =    28.058796382
 TOTAL ENERGY         =  -275.104224198

 @ End Energy calculation


 @ Begin Gradient Calculation

 ANALYTICAL GRADIENT: 
----------------------------------------
 COORDINATE    XYZ            GRADIENT
----------------------------------------
    1X    -6.4195304902    -0.0246366805
    1Y     1.5248601164     0.0000000002
    1Z     0.0000000000     0.0000000004
    2X    -4.6090103520     0.0242851057
    2Y     1.5248601164     0.0000000000
    2Z     0.0000000000    -0.0000000001
----------------------------------------

 @ End Gradient Calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      MG       0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      O        1.81052  1     0.00000  0      0.00000  0      1    0    0

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   MG             0.5296             0.3666
   O             -0.5296            -0.3666
   TOTAL        -0.0000            -0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
   -7.0303    0.0000    0.0000    7.0303

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     1.026111000( 28.82%)
| DFT GRID OPERATIONS =     0.018639000(  0.52%)
| TOTAL SCF TIME      =     2.106636000( 59.18%)
|       TOTAL OP TIME      =     2.064295000( 57.99%)
|             TOTAL 1e TIME      =     0.053260000(  1.50%)
|             TOTAL 2e TIME      =     1.418844000( 39.86%)
|             TOTAL EXC TIME     =     0.533460000( 14.99%)
|       TOTAL DII TIME      =     0.038781000(  1.09%)
|             TOTAL DIAG TIME    =     0.033244000(  0.93%)
| TOTAL GRADIENT TIME      =     0.407778000( 11.45%)
|       TOTAL 1e GRADIENT TIME      =     0.060816000( 1.74%)
|       TOTAL 2e GRADIENT TIME      =     0.267341000( 7.51%)
|       TOTAL EXC GRADIENT TIME     =     0.078369000(  2.20%)
| TOTAL TIME          =     3.559901000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  3.6 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Wed Mar 17 12:12:34 2021
